src/data/selectors/grades.js

```diff
@@ -187,7 +187,7 @@ export const allGrades = ({ grades: { results } }) => results;
  */
 export const bulkImportError = ({ grades: { bulkManagement } }) => (
   (!!bulkManagement && bulkManagement.errorMessages)
-    ? `Errors while processing: ${bulkManagement.errorMessages.join(', ')}`
+    ? `Errors while processing: ${bulkManagement.errorMessages.join(' ')}`
```

Comment:
See [referenced comment](https://github.com/edx/edx-bulk-grades/pull/87/files#r675705500), I think the separator should live here.
```suggestion
    ? `Errors while processing: ${bulkManagement.errorMessages.join('; ')}`
```

Reply:
I am okay with it. I was just trying to follow the one we all agree were all of them ended with a semi-colon